components/nds-filters/src/FilterSummary.js

```diff
@@ -92,56 +87,52 @@ function ResultsFilters({ filters }) {
 		</ul>
 	);
 }
+function ResultsSorting({ sorting }) {
+	if (!sorting.length) return null;
 
-function ResultsSorting({ active, inactive }) {
-	if (!inactive.length && !active) return null;
-	return (
-		<p className="filter-summary__sort hide-print">
-			{active && (
-				<>
-					<span>
-						<span className="visually-hidden">Sorted by</span> {active.label}
-						<span className="visually-hidden">.</span>
-					</span>{" "}
-					|
-				</>
-			)}
+	// Find default active value if it exists
+	let defaultValue = "";
+	for (let i = 0; i < sorting.length; i++) {
+		if (sorting[i].active === true) {
+			defaultValue = sorting[i].value;
+			break;
+		}
+	}
+
+	// Run callback function passed in to the selected sorting option
+	const handleChange = e => {
+		const callback = sorting[e.target.selectedIndex].callback;
+		if (typeof callback === "function") {
+			callback();
+		}
+	};
+
+	// Hide button via JS, so it's visible if JS is disabled
+	const buttonStyle = { display: "none" };
```

Comment:
This won't work for server rendered apps that rehydrate will it as it will render this `style` attribute server side so it will always be hidden. I think we probably want a `useEffect` to remove the button?